src/core_modules/capture-core-utils/rulesEngine/services/VariableService/VariableService.js

```diff
@@ -66,12 +67,17 @@ export class VariableService {
             OptionSetHelper.getName(optionSets[trackedEntityAttributes[trackedEntityAttributeId].optionSetId].options, value)
             : value;
     }
-    static dateUtils = getDateUtils(momentConverter);
+
+    static dateUtils: any;
```

Comment:
I think `Object` is a better type for `dateUtils`. WDYT?

Reply:
I looked up the difference between `Object` and `any`, but apparently they are the same thing in flow. The NOTE in the last section on [this page](https://flow.org/en/docs/types/objects/) says quite explicitly that it's better to keep `any` if the alternative is to change it into Object, actually.